String/Aufgabe4Anagram.java

```diff
@@ -0,0 +1,72 @@
+package QC_Java.String;
+public class Aufgabe4Anagram {
+    /*
+    Aufgabe 4
+    Ein Anagramm ist ein Satz, der durch Umstellen aller Buchstaben eines anderen
+    Satzes gebildet werden kann. Gross- und Kleinschreibung wird dabei ignoriert.
+    Ebenso spielen Leerzeichen keine Rolle. Aus dem Wort "Desperation" läßt sich zum
+    Beispiel der Satz "A Rope Ends It" bilden.
+    Implementieren eine statische Methode isAnagram, mit der zwei Zeichenketten
+    daraufhin überprüft werden können, ob sie ein Anagram bilden.
+    Verwenden Sie die folgende Signatur für die Methode:
+    public static boolean isAnagram(String str)
+     */
+    public static boolean isAnagram(){
```

Comment:
Method should accept 2 Strings, in your case it doesn't accept anything. Please re-implement your code to accept 2 strings and do a check.

Reply:
Fixed